src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -54,6 +56,7 @@ namespace Catalyst.Core.Modules.Consensus.Tests.UnitTests.Deltas
 {
     public sealed class DeltaBuilderTests
     {
+        private const ulong DeltaGasLimit = 8_000_000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private field 'DeltaGasLimit'.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)